git_machete/annotation.py

```diff
@@ -31,11 +33,17 @@ def __init__(self, annotation: str):
             self._push_text = 'push=no'
             self._annotation_without_qualifiers = re.sub(sub_pattern('push'), ' ', self._annotation_without_qualifiers)
 
+        slide_out_match = re.match(match_pattern('slide-out'), annotation)
+        if slide_out_match:
+            self.slide_out = False
+            self._slide_out_text = 'slide-out=no'
+            self._annotation_without_qualifiers = re.sub(sub_pattern('slide-out'), ' ', self._annotation_without_qualifiers)
+
     def get_annotation_text_without_qualifiers(self) -> str:
         return self._annotation_without_qualifiers.strip()
 
     def get_qualifiers_text(self) -> str:
-        return f'{self._rebase_text.strip()} {self._push_text.strip()}'.replace('  ', ' ').strip()
+        return f'{self._rebase_text.strip()} {self._push_text.strip()} {self._slide_out_text.strip()}'.replace('  ', ' ').strip()
```

Comment:
Actually... is there a need to strip any of these? given what values can be assigned to them

Reply:
Right, there is no need to strip these values since we are hard coding its values.